### DISCUSSION TITLE
Improve Largest Contentful Paint score on PageSpeed insight

### DISCUSSION THREAD

**Discussion by memegauste** [nolimits4web/swiper#4521] [Q&A]
Hiya,
So I want to ask you about providing improvements of performance to SwiperJS because of PageSpeed insights scores in LCP metric.
I use Swiper in image div section (that means the whole div got full width and about half of FullHD screen height). Scores I get are from 3 to 4.5 seconds range. 

![image](https://user-images.githubusercontent.com/21007545/116968592-136e8400-acb5-11eb-8d21-5a166cbc36aa.png)

The element that is affected (the most?) is:

![image](https://user-images.githubusercontent.com/21007545/116968562-03ef3b00-acb5-11eb-9037-fc0d3ad599f0.png)

```css
div.swiper-slide.swiper-lazy.home-slider-centered.swiper-slide-active.swiper-lazy-loaded
```
Basically I use lazyloading too for improvement of performance in my website.
I have also included the newest version of the package in files of the time of writing - `6.5.9`.

**Comment by shkipan** [comment]
@nolimits4web , are there any updates on this topic? I am still getting an LCP warning

**Comment by KubaDrx** [comment]
@nolimits4web can we get some updates, any hints on how this can be possibly improved?

**Comment by cernymatej** [comment]
Has anyone come up with a workaround for the bad lighthouse scores?

**Comment by conkonig** [comment]
Any updates on this? LCP scores with swiper are pretty horrendous..

**Comment by efimBistrov** [comment]
Any updates of this?
